app/src/components/CardButton.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import CardButton from './CardButton'
+
+import '@testing-library/jest-dom'
```

Comment:
My suggestion would be that third party imports should come before local ones. What do you think?

Reply:
I Agree.

BTW, shouldn't lint fix this? @eduardomrodrigues any thoughts on that?

Reply:
Great idea! Let me see what this Lint can do! I'm working on it